R/experiments.R

```diff
@@ -154,26 +148,25 @@ set_experiment_tag <- function(key, value, experiment_id, client) {
 #' active experiment if both `experiment_id` and `name` are unspecified.
 #'
 #' @param experiment_id ID of the experiment.
-#' @param name The experiment name. Only one of `name` or `experiment_id` should be specified.
+#' @param experiment_name The experiment name. Only one of `name` or `experiment_id` should be specified.
 #' @param client An MLFlow client. Defaults to `NULL` and will be auto-generated.
 #'
 #' @export
-get_experiment <- function(experiment_id, name, client) {
+get_experiment <- function(experiment_id = get_active_experiment_id(), experiment_name = NULL, client = mlflow_client()) {
```

Comment:
i like the change to `experiment_name`

Reply:
yeah I want to do a lot more of this in different places, but prob a future PR. there's stuff like `name`, `source`, `path` etc everywhere and they all mean different things so it's super confusing